### PR TITLE
Prepare renaming the master branch

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,22 +43,22 @@ composer create-project --no-install contao/managed-edition <directory> <branch>
 ```
 
 Replace `<directory>` with the directory you want to install the Managed
-Edition in (use `.` for the current one). Replace `<branch>` with `dev-master`
-if you want to add a new feature or with `<lts-version>.x-dev` (currently
-`4.9.x-dev`) if you want to fix a bug.
+Edition in (use `.` for the current one). Replace `<branch>` with `4.x` if you
```
Comment (aschempp) [inline README.md]: shouldn't that be `4.x-dev`?

File: README.md [hunk 2]
```diff
@@ -43,22 +43,22 @@ composer create-project --no-install contao/managed-edition <directory> <branch>
...
+if you want to fix a bug.
 
 Then adjust the `require` section in your `composer.json` file so Composer
 loads the monorepo instead of the individual bundles:
 
 ```json
 "require": {
     "php": "^7.2",
-    "contao/contao": "dev-master"
+    "contao/contao": "4.x"
```
Comment (aschempp) [inline README.md]: `4.x-dev`

**Review by aschempp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (leofeyer) [reply README.md]: Yes. Fixed in be86e327d6e30ac02ab02387c3bd94463c501e6c.

File: README.md (hunk 2 above)
Comment (leofeyer) [reply README.md]: Fixed in be86e327d6e30ac02ab02387c3bd94463c501e6c.

**Pull request merged**